src/util/get-electron-version.js

```diff
@@ -0,0 +1,7 @@
+import path from 'path';
+import readPackageJSON from './read-package-json';
+
+export default async (projectDir) => {
```

Comment:
@MarshallOfSound we're going to need to figure out how to merge this with https://github.com/electron-userland/electron-forge/pull/432/files#diff-f5dc721cced1da6c967b89a0309237de